include/worker.hpp

```diff
@@ -415,6 +415,36 @@ class Worker
      */
     void performBackupAndRestore(types::VPDMapVariant& io_srcVpdMap);
 
+    /**
+     * @brief API to update "Functional" property.
+     *
+     * The API sets the default value for "Functional" property once if the
+     * property is not yet populated over DBus. As the property value is not
+     * controlled by the VPD-Collection process, if it is found already
+     * populated, the functions skips re-populating the property so that already
+     * existing value can be retained.
+     *
+     * @param[in] i_inventoryObjPath - Inventory path as read from config JSON.
+     * @param[in] io_interfaces - Map to hold all the interfaces for the FRU.
+     */
+    void processOperationalInterface(const std::string& i_inventoryObjPath,
+                                     types::InterfaceMap& io_interfaces);
+
+    /**
+     * @brief API to update "enable" property.
+     *
+     * The API sets the default value for "enable" property once if the property
+     * is not yet populated over DBus. As the property value is not controlled
+     * by the VPD-Collection process, if it is found already populated, the
+     * functions skips re-populating the property so that already existing value
+     * can be retained.
+     *
+     * @param[in] i_inventoryObjPath - Inventory path as read from config JSON.
+     * @param[in] io_interfaces - Map to hold all the interfaces for the FRU.
+     */
+    void processEnableInterface(const std::string& i_inventoryObjPath,
```

Comment:
property ?

Reply:
the function is basically doing something with the property and not the interface so should the name of the function be likewise.

Reply:
the interface could be there but the check is for property and doing processing for it

Reply:
Ok will update that.